db/seeds.rb

```diff
@@ -4,8 +4,9 @@
 service = Service.find_by url: 'localhost'
 unless service
   service = FactoryGirl.create :service, name: 'dev stuff', url: 'localhost'
-  item_type = FactoryGirl.create :item_type, name: 'TEST_CREATE_RENTAL_TYPE', creator: service
-  item = FactoryGirl.create :item, name: 'TEST_CREATE_RENTAL_ITEM', reservable: true, item_type: item_type
+  item_type = FactoryGirl.create :item_type, name: 'TEST_ITEM_TYPE', creator: service
+  # now we're going to need a lot of items to test with
+  FactoryGirl.create :item, reservable: true, item_type: item_type until Item.count > 40
```

Comment:
Very minor stylistic opinion - I think it's easier to parse "do something 40 times" than "do this until there are 40 of something that is created in each iteration". That is, I might just use `40.times do` here.